chroot-bin/volume

```diff
@@ -0,0 +1,178 @@
+#!/bin/sh
```

Comment:
See installer/main.sh for what the head of scripts should look like (especially the copyright text; no need to copy the entire license into each script).

Benefit of putting the USAGE text in the way the installer does it is that you don't need to repeat yourself with a header comment.